<launch>
    <!-- The ros nodes -->
    <!-- <node pkg="norfair_ros" name="detection_visualizer" type="detection_visualize.py"   output="screen"/> -->
    <node pkg="norfair_ros" name="tracker"              type="tracker.py"               output="screen"/>
    <node pkg="norfair_ros" name="tracker_visualizer"   type="tracker_visualize.py"     output="screen"/>

    <!-- The parameter file -->
    <rosparam file="$(find norfair_ros)/config/norfair_compare.yaml"/>

</launch>